src/content/docs/style-guide/formatting/code-examples.mdx

```diff
@@ -127,7 +144,46 @@ To format inline code (like `maxSampleTimes` or `TransactionError`), surround th
 
 ## Multi-line code blocks [#code-block]
 
-To format one or more lines of code, insert three backticks ( ```` ``` ```` ) above and below your code. This essentially acts like the HTML `<pre>` tag.
+To format one or more lines of code, insert three backticks ( ```` ``` ```` ) above and below your code. This essentially acts like the HTML `<pre>` tag. 
+
+When you sandwich your code between three backticks, you need to consider both the code layout (indentation) and the syntax highlighting.
+
+### Code layout [#code-layout]
+
+As a technial writer you'll be given code snippets from a variety of languages. To make it easier for your readers to understand code snippets, make sure that lines are indented and grouped in a helpful way.
+
+#### Strict indentation [#strict]
+Some programming and markup languages require specific indentation (for example, Python and YAML). Check with your subject matter expert if you are unsure about whether the snippet you've been given needs to have specific indentation in order to work. You can always run the snippet through an online code prettifier if it gets jumbled when you transfer it from another editing tool.
+
+#### Non-strict indentation [#casual]
+For languages that are casual about indendataion, we recommend that you look for similar examples and follow those patterns. In the case of NRQL, which works with a wide range of indentation and grouping, we recommend you do the following to make it easier to read:
```

Comment:
Also, I think we should acknowledge that all these conventions are inspired by standard SQL formatting conventions